app/views/podcast_planner/_form_calculator.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with(url: podcast_planner_path(@podcast), method: :get, data: {controller: "click"}) do |form| %>
```

Comment:
Ported/modified from `_form_main.html.erb`.  Trying to clarify the purpose of this form a bit (to "calculate" the draft form).